src/m365/spo/commands/web/WebProperties.ts

```diff
@@ -1,3 +1,5 @@
+import { RoleAssignment } from "@microsoft/microsoft-graph-types";
```

Comment:
actually, shouldn't we use the `RoleAssignment` that you created not from `graph-types`